plugins.d/Mail_Relaying/mail_relay.py

```diff
@@ -83,27 +101,51 @@ def run():
 
         while 1:
             fields = [
-                ('Host', host, field_width, field_limit),
-                ('Port', port, field_width, field_limit),
-                ('Login', login, field_width, field_limit),
-                ('Password', password, field_width, field_limit)
+                ('Host', 1, 0, host, 1, 10, field_width, field_limit),
+                ('Port', 2, 0, port, 2, 10, field_width, field_limit),
+                ('Login', 3, 0, login, 3, 10, field_width, field_limit),
+                ('Password', 4, 0, password, 4, 10, field_width, field_limit)
             ]
 
             retcode, values = console.form(TITLE, FORMNOTE, fields)
+            host, port, login, password = tuple(values)
```

Comment:
does ``host, port, login, password = values`` not work here? Is there a reason to casting this as a tuple?

Reply:
Nope, just that's what it was before. I only changed the order (and perhaps the indent level too?). The line itself was already there...